torch_glow/src/PyTorchCommon.cpp

```diff
@@ -55,9 +77,13 @@ void registerGlowOp() {
   torch::jit::RegisterOperators op({torch::jit::Operator(
       getGlowSymbol(),
       [](const torch::jit::Node *node) {
-        return [node](torch::jit::Stack &stack) {
-          llvm::Error err =
-              CachingGraphRunner::getGraphRunner()->runGraph(node, stack);
+        std::shared_ptr<torch::jit::Graph> graph = node->g(at::attr::Subgraph);
```

Comment:
When is this shared_ptr released? Right after the op run is done?

Reply:
Once the OperationCreator is called, the Operation get's registered with the OperatorRegistry, but from reading that code I don't see any removal and [this comment ](https://github.com/pytorch/pytorch/blob/c6617b370b903b74a5647dd34e69fd0081713df8/torch/csrc/jit/operator.h#L35-L63)kind of suggests that things go in there forever so may be this does not go away when the graph is destroyed. If anyone knows of a better way to manage the lifetime of CachingGraphRunners let me know.

Reply:
Yeah, after reading the dispatching a bit I think this should work, as `std::shared_ptr<torch::jit::Graph> graph` created here will be persistent.